Jupyter Notebook Data Curation Primer/Jupyter Notebooks Data Curation Primer.md

```diff
@@ -118,25 +118,26 @@ The following elements outline recommendations for repositories accepting Jupyte
   - Jupyter implementation details
     - Jupyter version
     - Distribution (e.g. Anaconda)
-    - Kernel version
+    - Kernel version  (programming language plus version)
   - README
-    - Documents what the Jupyter Notebook is for
-    - Request that this file include citation(s) to third-party algorithms and analyses
-    - Recommend code comments within the Notebook file itself in addition to the README file
+    - Documents what the Jupyter Notebook is for (but recommendation is that the Notebook utilize code comments)
```

Comment:
Code comments should not be seen as a replacement or alternative to providing a README file. The code comments are used to describe what specific sets of cells do, but the notebook itself can have a much broader description and context.